src/shogun/kernel/ShiftInvariantKernel.h

```diff
@@ -58,6 +58,8 @@ class ShiftInvariantKernel: public Kernel
 	friend class internal::KernelManager;
 
 public:
+	using PRNG = std::mt19937_64;
```

Comment:
I wonder whether we can be less specific here? @vigsterkr @gf712

Reply:
Two cents, how about we template the class with PRNG?

Reply:
nono, I was more thinking about forward declaring it, the type is not really needed in the header ...